.github/workflows/ci.yaml

```diff
@@ -12,6 +12,54 @@ env:
   PLATFORMS: linux/amd64,linux/arm64
 
 jobs:
+  lint: 
+    runs-on: ubuntu-22.04
+    steps:
+      - name: "Checkout backend(REST)"
+        uses: actions/checkout@v4
+        with:
+          path: backend
+          fetch-depth: 0
+          fetch-tags: 1
+
+      - name: "lint"
+        run: |
+          cd backend
+
+          echo "Fix mess with tags in actions/checkout..."
+          git fetch -f && git fetch -f --tags
+
+          sudo apt-get install -y make
+          pip3 install -r requirements.txt
+          pip3 install -r requirements-dev.txt
+
+          #Run linter
+          make lint
+  
+  format:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: "Checkout backend(REST)"
+        uses: actions/checkout@v4
+        with:
+          path: backend
+          fetch-depth: 0
+          fetch-tags: 1
+
+      - name: "format"
+        run: |
+          cd backend
+
+          echo "Fix mess with tags in actions/checkout..."
+          git fetch -f && git fetch -f --tags
+
+          sudo apt-get install -y make
+          pip3 install -r requirements.txt
+          pip3 install -r requirements-dev.txt
+
+          #Run formatter
+          AUTOPEP8_EXIT_CODE=--exit-code make format-code
```

Comment:
I would leave this one as is, because the moment you enable the flag, the exit code of the autopep will be 2 when the code is formatted, potentially breaking any future integration such as pre-commit or other tools.

Reply:
ACK